back/api/models.py

```diff
@@ -19,8 +20,12 @@ def __str__(self):
 
 
 class Document(models.Model):
-    name = models.CharField(_('name'), max_length=80, blank=True)
-    link = models.CharField(_('link'), max_length=2000, blank=True)
+    name = models.CharField(_('name'), max_length=80)
+    link = models.URLField(
+        _('link'),
+        help_text="Veuillez compléter l'URL ci-dessus",
```

Comment:
There's no french in the code until now. As this is the API can we keep help_texts in english or with _('...')?